nni/algorithms/compression/pytorch/pruning/dependency_aware_pruner.py

```diff
@@ -82,17 +82,14 @@ def update_mask(self):
             self._dependency_update_mask()
 
     def validate_config(self, model, config_list):
-        schema = CompressorSchema([{
+        schema = PrunerSchema([{
             Optional('sparsity'): And(float, lambda n: 0 < n < 1),
             Optional('op_types'): ['Conv2d'],
             Optional('op_names'): [str],
             Optional('exclude'): bool
         }], model, logger)
 
         schema.validate(config_list)
-        for config in config_list:
-            if 'exclude' not in config and 'sparsity' not in config:
-                raise SchemaError('Either sparisty or exclude must be specified!')
```

Comment:
why remove this validation?

Reply:
this validation move to `PrunerSchema` and `QuantizerSchema`